File: public/docs/_examples/component-styles/ts/app/hero-details.component.ts
import { Component, Input } from '@angular/core';
import { Hero } from './hero';
import { HeroTeamComponent } from './hero-team.component';

// #docregion styleurls
@Component({
  selector: 'hero-details',
  template: `
    <h2>{{hero.name}}</h2>
    <hero-team [hero]=hero></hero-team>
    <ng-content></ng-content>
  `,
  styleUrls: ['app/hero-details.component.css'],
  directives: [HeroTeamComponent]
})
export class HeroDetailsComponent {
  // #enddocregion styleurls
  @Input() hero: Hero;
  // #docregion styleurls
}

File: public/docs/_examples/component-styles/dart/lib/hero_details_component.dart
import 'package:angular2/core.dart';
import 'hero.dart';
import 'hero_team_component.dart';

// #docregion styleurls
@Component(
    selector: 'hero-details',
    template: '''
      <h2>{{hero.name}}</h2>
      <hero-team [hero]=hero></hero-team>
      <ng-content></ng-content>''',
    styleUrls: const ['hero_details_component.css'],
    directives: const [HeroTeamComponent])
class HeroDetailsComponent {
  // #enddocregion styleurls
  @Input() Hero hero;
  // #docregion styleurls
}
